app/gui/view/component-browser/component-list-panel/grid/src/lib.rs

```diff
@@ -265,7 +265,7 @@ impl component::Model for Model {
         let enterable_elements = default();
         let colors = default();
         let requested_section_info = default();
-        let base_layer = &app.display.default_scene.layers.node_searcher_text;
```

Comment:
Why this name change? Did the logic of this layer change?

Reply:
This looked like a mistake. Using text layer for whole grid is surprising and different from what all demo scenes do. Individual sublayers created here are guaranteeing correct draw order anyway, including the dedicated text sublayer created in the grid-view.

With this change, `node_searcher_text` is only used by the old searcher, so it could be removed with it.